src/stageCache.js

```diff
@@ -137,15 +132,14 @@ const updateStageCacheSettings = async (settings, serverless) => {
     let endpointPatch = createPatchForEndpoint(endpointSettings, serverless);
     patchOps = patchOps.concat(endpointPatch);
   }
-  const apiGateway = new AWS.APIGateway();
   let params = {
     restApiId,
     stageName: settings.stage,
     patchOperations: patchOps
   }
 
   serverless.cli.log(`[serverless-api-gateway-caching] Updating API Gateway cache settings.`);
-  await apiGateway.updateStage(params).promise();
+  await serverless.providers.aws.request('APIGateway', 'updateStage', params, settings.stage, settings.region);
```

Comment:
@thebeatlecoder You probably need to do this in your plugin too. Serverless exposes its `aws` instance, that's what we should be using.